support-frontend/assets/helpers/user/user.js

```diff
@@ -139,9 +140,11 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
     dispatch(setFirstName(window.guardian.user.firstName));
     dispatch(setLastName(window.guardian.user.lastName));
     dispatch(setFullName(`${window.guardian.user.firstName} ${window.guardian.user.lastName}`));
+    // default value from Identity Billing Address, or Fastly GEO-IP
+    dispatch(setStateField(window.guardian.user.address4 || window.guardian.geoip.stateCode));
     dispatch(setIsSignedIn(true));
     dispatch(setEmailValidated(getEmailValidatedFromUserCookie(cookie.get('GU_U'))));
-  } else if (userAppearsLoggedIn) {
+  } else if (userAppearsLoggedIn) { // TODO - remove in another PR as this condition is deprecated
```

Comment:
Why is this condition deprecated? We are using it to surface a 'sign in' CTA on the thank you page

Reply:
It's the condition (if statement) based on this value, not the value that's deprecated, because routes.oneOffContribAutofill does not exist.

Reply:
Got it, thanks